docker/openproblems-python-extras/requirements.txt

```diff
@@ -6,4 +8,5 @@ umap-learn
 phate
 pyensembl
 pybedtools
+scvi-tools>=0.9.1
```

Comment:
@giovp looks like you're setting the requirement to `>=0.9.1` here. Is this the image you need `>=0.16` on?

Reply:
yes thank you!!!!

Reply:
fixed it!